methods.py

```diff
@@ -10,7 +10,7 @@
 from pathlib import Path
 from typing import Generator, Optional
 
-# Get the "Godot" folder name ahead of time
```

Comment:
This file also has comments changed.

Reply:
This comment is contextual and requires changing to blazium as there will be no Godot folder, other than that I will revert the rest of non-required comment and variable changes.
If you still insist I will change it.

Reply:
This one is fine then, thanks!

Reply:
fixed